### ISSUE TITLE
[Accessibility]On Windows high contrast themes, disabled toolStripMenuItem should have disabled checkmark when enabled 'Checked' property 

### ISSUE THREAD

**Issue by Olina-Zhang** [dotnet/winforms#7980]
### .NET version

8.0.100-alpha.1.22514.14 

### Did it work in .NET Framework?

No

### Did it work in any of the earlier releases of .NET Core or .NET 5+?

No, it also repro in previous .Net versions

### Issue description

On Windows high contrast themes, disabled toolStripMenuItem doesn't have disabled checkmark when enabled 'Checked' property 

**Windows default theme**, it is correct:
![image](https://user-images.githubusercontent.com/26474449/197093467-709cf744-9fb0-41d4-85e7-ce19c0898a68.png)

**Windows 10 high contrast themes:** checkmark icon is not disabled when its toolStripItem "Options" is disabled, it has same icon as toolStripItem is not disabled, see comparison in https://github.com/dotnet/winforms/pull/7971#issuecomment-1285723449
![image](https://user-images.githubusercontent.com/26474449/197093595-0200bb5f-ee4f-4969-9f1d-406fe57d5389.png)

### Steps to reproduce

1. Use attached application: 
[WinFormsApp75.zip](https://github.com/dotnet/winforms/files/9835150/WinFormsApp75.zip)
2. Set windows to different HC themes
3. build and run application
4. Observe the checkmark icon for disabled toolStripMenuItem "Options_Disabled"

**Comment by merriemcgaw** [comment]
@Olina-Zhang what does it look like on Win11? Is it good enough there?

**Comment by Olina-Zhang** [comment]
No, it also has this issue on Win11 high contrast themes, the screenshot is in https://github.com/dotnet/winforms/pull/7971#issuecomment-1285723449.
